docs/source/performance/cache-backends.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Configuring cache backends
+description: How to configure Apollo Server's cache
+---
+
+# Configuring cache backends
+
+## Introduction
+
+Several features of Apollo Server (such as [automatic persisted queries](./performance/apq), the [response cache plugin](./performance/caching#caching-with-responsecacheplugin-advanced), and [`RESTDataSource`](./data/data-sources#restdatasource-reference)) take advantage of a cache backend. By default, Apollo Server uses a cache in memory but this can be configured to use a different backend such as Redis or Memcached.
+
+You can specify a cache backend using the `cache` option to the Apollo Server constructor. The cache backend must implement the [`KeyValueCache`](https://github.com/apollographql/apollo-utils/tree/main/packages/keyValueCache#keyvaluecache-interface) interface from the `@apollo/utils.keyvaluecache` package. Apollo maintains an `InMemoryLRUCache` implementation in that package as well as a [`KeyvAdapter`](https://github.com/apollographql/apollo-utils/tree/main/packages/keyvAdapter#keyvadapter-class) class from the `@apollo/utils.keyvadapter` package for wrapping the [`keyv`](https://www.npmjs.com/package/keyv) package which implements a number of useful cache backends.
+
+Unfortunately, the default configuration of Apollo Server exposes you to denial of service attacks.
+This is because APQs are enabled by default and the default cache in Apollo Server 3 is unbounded, meaning an attacker can exhaust your memory and crash your server. The default cache in Apollo Server 4 will be bounded; **we recommend opting in to that behavior by providing the `cache: "bounded"` option to your Apollo Server constructor or configuring the `cache` yourself.**
+
+## Configuring in-memory caching
+
+By default, Apollo Server's caching features use an unbounded cache. This is not safe for production use. If you'd like to configure the in-memory cache, Apollo provides the [`InMemoryLRUCache`](https://github.com/apollographql/apollo-utils/tree/main/packages/keyValueCache#inmemorylrucache) class from the `@apollo/utils.keyvaluecache` package. This class is a wrapper around the `lru-cache` package with a default maximum of approximately 30MiB of memory. The `InMemoryLRUCache` is configurable with the options from `lru-cache`. See the [`lru-cache` documentation`](https://www.npmjs.com/package/lru-cache) for more information on configuration options.
```

Comment:
How would you feel about splitting this up into two paragraphs? Something like this? 
```suggestion
**Apollo Server's default caching features use an unbounded cache, which is not safe for production use.** If you'd like to configure the in-memory cache, Apollo provides the [`InMemoryLRUCache`](https://github.com/apollographql/apollo-utils/tree/main/packages/keyValueCache#inmemorylrucache) class from the `@apollo/utils.keyvaluecache` package. 

The `InMemoryLRUCache` class is a wrapper around the `lru-cache` package and has a default maximum of approximately 30MiB of memory. You can configure an instance of `InMemoryLRUCache` with the same options as found in the `lru-cache` package, see the [`lru-cache` documentation](https://www.npmjs.com/package/lru-cache) for more details. 
```

Reply:
I feel like you should probably specifically say something along the lines of "configuring your cache will make it safe to use in production", maybe adding a specific callout to `cache: "bounded"` in this section as well?